Header header
uint8 seed_num
uint8 status
---
bool result 